.github/workflows/create-release.yml

```diff
@@ -25,7 +25,8 @@ jobs:
       - uses: actions/setup-node@v3
         with:
           node-version: 16
-      - run: npm ci
+        # There is no `package-lock.json` file
```

Comment:
Why is this [different from the pip integration](https://github.com/dequelabs/axe-devtools-pip/blob/develop/package-lock.json)?

Reply:
https://github.com/dequelabs/axe-core-gems/pull/174

Reply:
Gotcha. This way actually makes _more_ sense.